src/robusta/core/playbooks/playbooks_event_handler_impl.py

```diff
@@ -336,6 +337,11 @@ def is_healthy(
 
     def handle_sigint(self, sig, frame):
         logging.info("SIGINT handler called")
+
         if not self.is_healthy():  # dump stuck trace only when the runner is unhealthy
             StackTracer.dump()
+
+        for sink in self.registry.get_sinks().get_sinks_by_type(RobustaSink):
+            sink.dal.set_cluster_active(False)
```

Comment:
I would add this function in SinkBase
```
  def set_cluster_active(self, is_active: bool):
      pass
```
   and then in robusta_sink
```
    def set_cluster_active(self, is_active: bool):
      self.dal.set_cluster_active(is_active)
```
and then you don't need the function `get_sinks_by_type`, 
and then if we want to signal it in any other sink we can (like in slack)